### DISCUSSION TITLE
Anyone willing to help me to set Copilot.lua and make it work with cmp?

### DISCUSSION THREAD

**Discussion by StefanoDalMas** [NvChad/NvChad#1958] [Q&A]
Hi guys, I'm a complete newbie on vim and I'm trying to make copilot work with cmp.
I've tried but I don't know where to put anything because NVChad has files scattered across various directories and I don't know where to look for stuff... Can anyone help? I'd like to have in all files the corresponding LSP like clangd, rust-analyzer, pyright and so on with the cmp coming at me with the suggestion both from the LSP and from Copilot. 
Is this manageable?
Or maybe using the copilot.lua with the standard "ghost code" like in VSCode if that is faster, I don't know to be honest.
Thanks for the responses :)

**Comment by siduck** [comment]
hi @StefanoDalMas , I dont use copilot so cant help about it much! but check our discord server, many use it there! Would help you :D

**Comment by luizkowalski** [comment]
Hey @StefanoDalMas, check this discussion: https://github.com/NvChad/NvChad/discussions/1295
